website/docs/install-deploy/deploying-with-helm.md

```diff
@@ -0,0 +1,405 @@
+| `configurationOverrides.data.dir` | Local data directory | `/tmp/fluss/data` |
+| `configurationOverrides.internal.listener.name` | Internal listener name | `INTERNAL` |
+
+### Persistence Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `persistence.enabled` | Enable persistent volume claims | `false` |
+| `persistence.size` | Persistent volume size | `1Gi` |
+| `persistence.storageClass` | Storage class name | `nil` (uses default) |
+
+### Resource Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `resources.coordinatorServer.requests.cpu` | CPU requests for coordinator | Not set |
+| `resources.coordinatorServer.requests.memory` | Memory requests for coordinator | Not set |
+| `resources.coordinatorServer.limits.cpu` | CPU limits for coordinator | Not set |
+| `resources.coordinatorServer.limits.memory` | Memory limits for coordinator | Not set |
+| `resources.tabletServer.requests.cpu` | CPU requests for tablet servers | Not set |
+| `resources.tabletServer.requests.memory` | Memory requests for tablet servers | Not set |
+| `resources.tabletServer.limits.cpu` | CPU limits for tablet servers | Not set |
+| `resources.tabletServer.limits.memory` | Memory limits for tablet servers | Not set |
+
+
+## Advanced Configuration
+
+### Custom ZooKeeper Configuration
+
+For external ZooKeeper clusters:
+
+```yaml
+configurationOverrides:
+  zookeeper.address: "zk1.example.com:2181,zk2.example.com:2181,zk3.example.com:2181"
+  zookeeper.path.root: "/my-fluss-cluster"
+```
+
+### Network Configuration
+
+The chart automatically configures listeners for internal cluster communication and external client access:
+
+- **Internal Port (9123)**: Used for inter-service communication within the cluster
+- **External Port (9124)**: Used for client connections
+
+Custom listener configuration:
+
+```yaml
+appConfig:
+  internalPort: 9123
+  externalPort: 9124
+
+configurationOverrides:
+  bind.listeners: "INTERNAL://0.0.0.0:9123,CLIENT://0.0.0.0:9124"
+  advertised.listeners: "CLIENT://my-cluster.example.com:9124"
+```
+
+### Storage Configuration
+
+Configure different storage backends:
+
+```yaml
+configurationOverrides:
+  data.dir: "/data/fluss"
+  remote.data.dir: "s3://my-bucket/fluss-data"
+```
+
+## Upgrading
+
+### Upgrade the Chart
+
+```bash
+# Upgrade to a newer chart version
+helm upgrade fluss ./docker/helm
+
+# Upgrade with new configuration
+helm upgrade fluss ./docker/helm -f values-new.yaml
+```
+
+### Rolling Updates
+
+The StatefulSets support rolling updates. When you update the configuration, pods will be restarted one by one to maintain availability.
+
+## Custom Container Images
+
+### Building Custom Images
+
+To build and use custom Fluss images:
+
+1. Build the project with Maven:
+```bash
+mvn clean package -DskipTests
+```
+
+2. Build the Docker image:
+```bash
+# Copy build artifacts
+cp -r build-target/* docker/build-target/
```

Comment:
should be `cp -r build-target/* docker/fluss/build-target`, the fluss dockerfile has been moved to `docker/fluss`.